shared/js/dataDriver.js

```diff
@@ -277,13 +292,15 @@ loop.DataDriver = function() {
      * @param {String} token The room token to use for connections.
      */
     _connectToRoom(token) {
+      this._retryTimeout = this.INITIAL_RETRY_TIMEOUT;
       this._roomToken = token;
       this._sse = new EventSource(this._buildUrl() + "?" + this._buildQuery({
         limitToLast: 1,
         orderBy: '"timestamp"'
       }));
       this._sse.addEventListener("open", this);
       this._sse.addEventListener("put", this);
+      this._sse.addEventListener("error", this);
```

Comment:
Whoops, you're right about the exponential backoff being reset improperly. I added the code that reset it after I tested it. Fixing.

Yes, error happens for the initial connection as well.